src/components/BlogCard/index.js

```diff
@@ -21,56 +36,100 @@ const stripMarkdown = (markdown) => {
-    border={{
-      side: 'bottom',
-      color: 'light-2',
-      size: 'medium',
+const columns = {
+  small: ['auto'],
+  medium: ['auto', 'auto'],
+  large: ['auto', 'auto', 'auto', 'auto'],
+  xlarge: ['auto', 'auto', 'auto', 'auto'],
+};
+
+const rows = {
+  small: ['auto', 'auto', 'auto'],
+  medium: ['auto', 'auto'],
+  large: ['auto'],
+  xlarge: ['auto'],
+};
+
+export const AllBlogsCard = ({
+  children,
+  overrideColumns,
+  overrideRows,
+  areas,
+  ...props
+}) => (
+  <ResponsiveContext.Consumer>
+    {(size) => {
+      // Take into consideration if not array is sent but a simple string
+      let columnsVal = columns;
+      if (columns) {
+        if (columns[size]) {
+          columnsVal = columns[size];
+        }
+      }
+
+      let rowsVal = rows;
+      if (rows) {
+        if (rows[size]) {
+          rowsVal = rows[size];
+        }
+      }
+
+      // Also if areas is a simple array not an object of arrays for
+      // different sizes
+      let areasVal = areas;
+      if (areas && !Array.isArray(areas)) areasVal = areas[size];
+
+      return (
+        <Grid
+          {...props}
+          areas={!areasVal ? undefined : areasVal}
+          rows={!rowsVal ? size : rowsVal}
+          columns={!columnsVal ? size : columnsVal}
+        >
+          {children}
+        </Grid>
+      );
     }}
-    justify="center"
+  </ResponsiveContext.Consumer>
+);
+
+AllBlogsCard.propTypes = {
+  overrideColumns: PropTypes.string,
+  children: PropTypes.node,
+  overrideRows: PropTypes.string,
+  areas: PropTypes.string,
+};
+
+export const BlogCard = ({ node, ...rest }) => (
+  <GrommetCard
+    pad="large"
+    direction="row"
+    justify="between"
     {...rest}
+    elevation="medium"
+    wrap
   >
-    <Box>
-      {node.frontmatter.date && node.frontmatter.author && (
-        <Markdown>
-          {`${dateFormat.format(new Date(node.frontmatter.date))} by **${
-            node.frontmatter.author
-          }**`}
-        </Markdown>
-      )}
-      <Heading
-        level={2}
-        size="large"
-        margin={{ top: 'none', bottom: 'xsmall' }}
-      >
-        {node.frontmatter.title}
-      </Heading>
-      {node.frontmatter.version && (
-        <Text size="small" color="neutral-4">
-          {node.frontmatter.version}
-        </Text>
-      )}
-    </Box>
-    <Box width="large">
-      <Paragraph margin="none">
-        {node.frontmatter.description || stripMarkdown(node.excerpt)}
-      </Paragraph>
-    </Box>
-    <Link
-      to={`/${node.fields.sourceInstanceName}${node.fields.slug}`}
-      label="Read more >"
-    />
-  </Box>
+    <NavLink to={`${node.path || node.frontmatter.path}`}>
+      <Box gap="small">
+        <Box align="start">
+          <Image src="/img/blogs/Avatar1.svg" />
+        </Box>
+        <Box align="start">
+          <Text color="text">{node.author || node.frontmatter.author}</Text>
+        </Box>
+        <Heading level={4} margin="none" color="text">
```

Comment:
why specify the color here?

Reply:
Gatsby link default color was applied to the text so, I have used inline color but now, instead of Navlink I used onclick navigate(link) which fixed the color issue.